src/checks/leading_character.rs

```diff
@@ -2,13 +2,13 @@ use crate::checks::Check;
 use crate::common::*;
 
 pub(crate) struct LeadingCharacterChecker {
-    template: String,
+    template: &'static str,
 }
 
 impl Default for LeadingCharacterChecker {
     fn default() -> Self {
         Self {
-            template: String::from("Invalid leading character detected"),
+            template: "LeadingCharacter: Invalid leading character detected",
```

Comment:
In the future I would like to have the ability to disable some checks by their names.
To do that, all checks should have the name field.
Please, move the check's name from the template to the special field.
Also, I would like to have a method to show the check's name in the message, for example:
https://github.com/mgrachev/dotenv-linter/commit/38b9558bd38341ff13fa07f27fc069a9ce9aa48e#diff-b598dd141594bc9ca17caeca0ed82794R10